docs/getting-started/enterprise/splunk-app.md

```diff
@@ -0,0 +1,104 @@
+# Splunk App
+
+The Bitwarden Splunk app fetches event log data from the Bitwarden Public API and makes it available
+in Splunk.
+
+## Requirements
+
+- Docker. If you're using an Apple Silicon Mac, enable _Docker Desktop_ -> _Settings_ -> _General_
+  -> _Use Rosetta for x86_64/amd64 emulation on Apple Silicon_
+- Python 3.8 or 3.9
+- [Poetry][poetry]
+- libmagic (macOS only), available via homebrew: `brew install libmagic`
+- A Bitwarden server with event logging configured and an enterprise organization. To set this up
+  locally, see the [Setup Guide](../server/guide.md) and [Event Logging](../server/events.md)
+
+## Set up and configuration
+
+### Configure your environment
+
+1. Clone the Github repository:
+
+   ```
+   git clone https://github.com/bitwarden/splunk.git
+   ```
+
+2. Navigate to the root of the repository:
+
+   ```
+   cd splunk
+   ```
+
+3. Activate the poetry shell:
+
+   ```
+   poetry shell
+   ```
+
+4. Tell poetry to use the required Python version:
+
+   ```
+   poetry env use <executable>
+   ```
+
+   Where `<executable>` is the executable for Python 3.8 or 3.9. If this is in your PATH variable
+   then you do not need to specify the full path. e.g. `poetry env use python3.8`
+
```

Comment:
i just did a fresh test and i think this should be the other way around.
We firstly tell poetry to create the virtual environment for the python version we want to use.
And then we activate it with `poetry shell`